apps/src/applab/DesignModeHeaders.jsx

```diff
@@ -17,6 +19,47 @@ const DesignModeHeaders = React.createClass({
     this.props.onToggleToolbox();
   },
 
+  chevronStyle(collapse) {
+    const style = {
+      display: 'inline-block',
+      position: 'absolute',
+      top: 0,
+      left: 8,
+      lineHeight: '30px',
+      fontSize: 18,
+      cursor: 'pointer',
+      color: this.props.isRunning ? color.dark_charcoal : color.lighter_purple,
+      ':hover': {
+        color: color.white,
+      },
+    };
+
+    if (collapse) {
+      style.transform = 'scale(-1, 1)';
+    }
+
+    return style;
+  },
+
+  hideToolboxIcon() {
```

Comment:
At first I thought these names were a little confusing because they seem like verbs, but I think it makes sense with the toggle function.